CIFAR10/train/fastai_pytorch.json

```diff
@@ -3,16 +3,16 @@
     "author": "fast.ai + students team: Jeremy Howard, Andrew Shaw, Brett Koonce, Sylvain Gugger",
     "authorEmail": "j@fast.ai",
     "framework": "fastai / pytorch",
-    "codeURL": "https://github.com/fastai/imagenet-fast/commit/2748de3b0e28ecca1e02d90b2d5bf7d9b9e58e3d",
+    "codeURL": "https://github.com/fastai/imagenet-fast/commit/1bc5aeec765572397c0ebe4a5d616d03beeeeec1",
     "model": "Custom Wide Resnet",
     "hardware": "8 * V100 (AWS p3.16xlarge)",
     "costPerHour": 24.48,
     "timestamp": "2018-04-18",
     "misc": {
-        "model implementation": "https://github.com/fastai/imagenet-fast/blob/2748de3b0e28ecca1e02d90b2d5bf7d9b9e58e3d/cifar10/models/wideresnet.py#L60",
-        "1cycle parameters": "cycles: 50, learning-rates: 50,15, cyclical max/min momentum: 0.95,0.85",
+        "model implementation": "https://github.com/fastai/imagenet-fast/blob/e93593ca0a7e226141050cbd55622ff08b0632c3/cifar10/models/wideresnet.py#L81",
```

Comment:
The commit hash in this link doesn't match the hash in the `codeURL` above. Were there any significant changes to the code or scripts you ran? I glanced over the commit history of your repo, and it looks like the changes between these two commits were for documentation. I just want to confirm.

Reply:
just linking to the docs update.  the last commit on master was something unrelated and thought this would be more relevant.  can switch if you want, but the models are the same.

Reply:
Thanks! That seems reasonable to me.